e2e/testdata/hydration/helm-components/kustomization.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Google LLC
+# Copyright 2025 Google LLC
```

Comment:
The copyright year also should not change here (please check other references as well)